lance-spark-base_2.12/src/main/java/com/lancedb/lance/spark/LanceConstant.java

```diff
@@ -14,6 +14,7 @@
 package com.lancedb.lance.spark;
 
 public class LanceConstant {
+  public static final String SEGMENT_ID = "_segmentid";
```

Comment:
Looks like this should be named like `FRAGMENT_ID = "_fragid"`? But given we already have `_rowaddr`, why do we need a separated fragment ID metadata column?